content/en/cloud_cost_management/container_cost_allocation.md

```diff
@@ -16,79 +16,93 @@ further_reading:
+- The full list of limitations can be found on the [GKE docs](https://cloud.google.com/kubernetes-engine/docs/how-to/cost-allocations#limitations)
+
+GKE cost allocation can be enabled by following the [instructions in the GKE docs](https://cloud.google.com/kubernetes-engine/docs/how-to/cost-allocations#enable_breakdown)
+
+
+### AWS ECS on EC2
 
 For ECS allocation, Datadog determines which tasks ran on each EC2 instance used for ECS. If you enable AWS Split Cost Allocation, the metrics allocate ECS costs by usage instead of reservation, providing more granular detail.
 
 Based on resources the task has used, Datadog assigns the appropriate portion of the instance's compute cost to that task. The calculated cost is enriched with all of the task's tags and all of the container tags (except container names) running in the task.
 
-### ECS on Fargate
+### AWS ECS on Fargate
 
 ECS tasks that run on Fargate are already fully allocated in the CUR. CCM enriches that data by adding out-of-the-box tags and container tags to the AWS Fargate cost.
 
 ### Everything else
 
-The information on this page pertains to EC2 compute and EBS costs for EC2 instances hosting Kubernetes pods or ECS tasks. All other costs are given the same value and tags as the source metric, `aws.cost.amortized`.
+The information on this page pertains to compute and AWS EBS costs for host instances running Kubernetes pods or ECS tasks. All other costs are given the same value and tags as the source metric, `aws.cost.amortized` or `gcp.cost.amortized`.
 
 
 ## Understanding spend
 
-Using the `allocated_spend_type` tag, you can visualize the spend category associated with your cost at the Kubernetes node, ECS host, EBS volume, or cluster level. Costs are allocated into different spend types:
+Using the `allocated_spend_type` tag, you can visualize the spend category associated with your cost at the Kubernetes node, AWS ECS host, AWS EBS volume, or cluster level. Costs are allocated into different spend types:
 
 | Spend type | Description    |
 | -----------| -----------    |
-| Managed service fee | Cost of associated fees charged by the provider for managing the cluster, such as EKS fees. |
+| Managed service fee | Cost of associated fees charged by the provider for managing the cluster, such as EKS/GKE fees. |
 | Usage | Cost of resources used by workloads. |
 | Workload idle | Cost of resources that are reserved and allocated but not used by workloads. |
 | Cluster idle | Cost of resources that are not reserved by workloads in a cluster. |
 
 ### Compute
 
-The cost of an EC2 instance is split into two components: 60% for the CPU and 40% for the memory. Each component is allocated to individual workloads based on their resource reservations and usage.
+ The cost of a host instance is split into two components: 60% for the CPU and 40% for the memory. Each component is allocated to individual workloads based on their resource reservations and usage.
```

Comment:
Should we somehow explain that Google compute costs are sometimes already split, and not by these percentages?
Edit: i see it's below, but might be nice to add something here too.

Reply:
```suggestion
The cost of a host instance is split into two components: 60% for the CPU and 40% for the memory. Each component is allocated to individual workloads based on their resource reservations and usage.
```